### ISSUE TITLE
Bug in AICOM Analyze

### ISSUE THREAD

**Issue by Igneous01** [KaukausInsurgency/kaukasusinsurgency-dcs#181]
This code here is wrong, Red should not be the 4th most expensive option. It should be that reinforcing is the most expensive option.

```Lua
if _own == "Neutral" then
        _cost = _cost + 1
        _type = AICOM.Enum.Actions.Attack
********** IF OWN BLUE, IT SHOULD BE REINFORCE ***********
      elseif _own == "Blue" then
        _cost = _cost + 5
        _type = AICOM.Enum.Actions.Attack
      elseif _own == "Red" then
********** IF OWN IS RED, IT SHOULD BE ATTACK **********
        _cost = _cost + 7
        _type = AICOM.Enum.Actions.Reinforce
      else
        -- contested
        _cost = _cost + 3
        _type = AICOM.Enum.Actions.Reinforce
      end
```

**Comment by Igneous01** [comment]
Currently this behavior is actually not too bad - it makes the AI more defensive in nature, opting to take neutral points and reinforcing existing ones. Maybe make this a configurable setting? (IE. Make AI behave more defensive, or offensive based on some true/false flag)

**Comment by Igneous01** [comment]
Fixed in aa1f4d87474a679fe0b702df0aee1108fc2142ef

**Comment by Igneous01** [comment]
What was fixed was that ai would attack its own zones, and reinforce enemy zones (so more aa was spawned into enemy zones which makes no sense)
